api/filetypes.json

```diff
@@ -4,8 +4,11 @@
     "dicom":        [ ".dcm", ".dcm.zip", ".dicom.zip" ],
     "parrec":       [ ".parrec.zip", ".par-rec.zip" ],
     "gephysio":     [ ".gephysio.zip" ],
+    "log":          [ ".log" ],
```

Comment:
It looks like the format here is to keep research/neuro file types up top, and more general file types below. Mind moving log down to the other list? Other than that, LGTM!